### PR TITLE
add ore id support for native

### LINKED ISSUE #579: Add ORE ID support for native
## Overview
Implement ORE ID ual to implement telos cloud wallet for native users

## Links
https://docs.oreid.io/ore-id/build/integration/ual
https://docs.oreid.io/ore-id/blockchains/eosio/get-eos-account-info-and-balance

## Tasks

- [x] install and implement oreid ual package for native telos cloud wallet
- [x] generate app id (if required)

## Testing
testnet
- [x] account creation
- [x] send transaction
- [x] buy resources
- [x] send to evm

mainnet
- [x] account creation*
- [x] send transaction
- [x] buy resources
- [x] staking
- [x] generate linked evm adress 
- [x] send to evm
- [x] withdraw from evm to native

### PR DESCRIPTION
**Pull request by donnyquixotic**
# Fixes #579 

## Description
- see issue
- minor refactoring & native error fixes

## Test scenarios
- see issue

## Checklist:

-   [x] I have performed a self-review of my own code
-   [x] I have commented my code, particularly in hard-to-understand areas
-   [x] I have cleaned up the code in the areas my change touches
-   [x] My changes generate no new warnings
-   [x] Any dependent changes have been merged and published in downstream modules
-   [x] I have checked my code and corrected any misspellings
-   [x] I have removed any unnecessary console messages
-   [x] I have tested for mobile functionality and responsiveness

### CODE REVIEW

**Comment by ezra-guzman** [comment]
when i attempt to use greymass fuel, i get an error:


https://github.com/telosnetwork/telos-wallet/assets/11837820/548efc6e-df72-43c9-998f-b10663537eaa

File: src/antelope/stores/account.ts [hunk 1]
```diff
@@ -256,7 +257,7 @@ export const useAccountStore = defineStore(store_name, {
                             network,
                         });
                     } else {
-                        const authenticator = getAntelope().wallets.getAutenticator(autoLogin)?.newInstance(label);
+                        const authenticator = getAntelope().wallets.getAuthenticator(autoLogin)?.newInstance(label);
```
Comment (Viterbo) [inline src/antelope/stores/account.ts:260]: oopss!! :see_no_evil: that was me. Thanks! :+1:

**Review by Viterbo — APPROVED** [review]
Dude, great job!! everything worked perfect!!
I tested: send, stake, profile, buy ram, all data fetched.

My only concern would be that we are calling "Telos Cloud Wallet" on EVM but "Ore ID Wallet" on native. Look:

image

File: src/antelope/stores/account.ts (hunk 1 above)
Comment (karynemayer) [reply src/antelope/stores/account.ts:260]: Telos cloud wallet logo

cloud-wallet-icon

**Comment by pmjanus** [comment]
1. Change name to Telos Cloud Wallet and use Logo from Karyne
2. Do not show Greymass popup

**Review by pmjanus — APPROVED** [review]

**Pull request merged**